src/node-lib/curriculum-api-2023/helpers/index.ts

```diff
@@ -0,0 +1,25 @@
+import { TierSchema } from "../queries/unitListing/unitListing.schema";
+
+export const toSentenceCase = (str: string) => {
+  return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
+};
+
+const hasCoreTierSet = new Set(["maths-ks4"]);
```

Comment:
just noting that when we make the new MV we should look into refactoring the unit listing page so we can determine this based on content instead (like the specialist unit listing page does)